database/schema.ts

```diff
@@ -292,6 +292,38 @@ export const purchasesTable = mysqlTable('purchases', {
     .notNull(),
 });
 
+export const bannerJobTable = mysqlTable('banner_jobs', {
+  id: bigint('id', {
+    unsigned: true,
+    mode: 'number',
+  })
+    .autoincrement()
+    .primaryKey(),
```

Comment:
think there's a nice alias for all this called `serial`

Reply:
Encountering a syntax issue and it's likely related to [this fresh bug](https://github.com/drizzle-team/drizzle-orm/issues/3333). Gonna circumvent this for now.

Reply:
Ah, wait, even going back on this change, the migrate is still bugging. Hm.

Reply:
Okay, handling and migrating your schema suggestions one at a time it appears `serial` alone is in fact the problem.

Setting `id` to `serial('id')` as per the [doc example](https://orm.drizzle.team/docs/column-types/mysql#serial) yields an error on migrate: `Incorrect table definition; there can be only one auto column and it must be defined as a key`

Adding `.primaryKey()` yields a different parse error: `You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'AUTO_INCREMENT NOT NULL' at line 1` with the highlighted sql being `ALTER TABLE `banner_jobs` MODIFY COLUMN `id` serial AUTO_INCREMENT NOT NULL;`

If it seems like I'm blindly cobbling things together it's likely because I am ^^'